client/ayon_blender/plugins/create/create_workfile.py

```diff
@@ -112,9 +115,8 @@ def collect_instances(self):
         # Add instance to create context
         self._add_instance_to_context(instance)
 
-    def remove_instances(self, instances):
-        for instance in instances:
-            node = instance.transient_data["instance_node"]
-            del node[AYON_PROPERTY]
-
-            self._remove_instance_from_context(instance)
+    def _find_old_workfile_property(self):
+        instance_node = bpy.data.collections.get(AYON_CONTAINERS)
+        if instance_node:
+            return instance_node.get(AYON_PROPERTY)
```

Comment:
This doesn't suffice, because in `collect` we still wouldn't find the `AYON_INSTANCES` collection on that _first_ collect in a legacy scene. We must consider this old one the actual instance node as well ... or somehow enforce moving the property over to the `AYON_INSTANCES` (and ensure it's existence).

Reply:
If we dive into the develop branch and test with the workfile, and we should find out that the instance data for workfile is not imprinted into `AYON_CONTAINERS`. I am thinking if it is necessity for implementing the backward compatibility for containers as it does not really imprint the data at all.

Reply:
It does, as soon as you save/publish once in publisher UI - it does not on just "reset" or open of the publisher UI however.

<img width="1033" height="982" alt="image" src="https://github.com/user-attachments/assets/ba47f9b5-7d91-4eae-9e0d-b9af66e70cd7" />